solutions/standard/variables.tf

```diff
@@ -380,3 +380,18 @@ variable "kibana_image_digest" {
 
 
 }
+variable "kibana_image_port" {
+  description = "Specify the port number used to connect to the Kibana service exposed by the container image. Default port is 5601 and it is only applicable if `enable_kibana_dashboard` is true"
+  type        = number
+  default     = 5601
+}
+
+variable "kibana_visibility" {
+  description = "Specify the visibility of Kibana application in order to define which endpoint is available for receiving the requests. Valid values are 'local_public', 'local_private' and 'local' and it is only applicable if `enable_kibana_dashboard` is true. See https://cloud.ibm.com/docs/codeengine?topic=codeengine-application-workloads#optionsvisibility. If Kibana is enabled, you can access the Kibana application over a IBM private network using the method outlined here https://cloud.ibm.com/docs/codeengine?topic=codeengine-vpe."
```

Comment:
I suggest to link to our markdown doc only, and in that markdown content, include those IBM Cloud links. Otherwise the variable description will be too long. Use the syntax `[Learn more](..link to markdown..)`